core/src/impl/Kokkos_Serial.cpp

```diff
@@ -155,9 +157,8 @@ void Serial::initialize( unsigned threads_count
 
   // Init the array of locks used for arbitrarily sized atomics
   Impl::init_lock_array_host_space();
-  #if defined(KOKKOS_ENABLE_PROFILING)
```

Comment:
Put that back in until we make the ExecSpace::initialize functions impl details.